core/modules/layout/js/grid-fallback.js

```diff
@@ -17,7 +17,9 @@
 Backdrop.behaviors.gridFloatFallback = {
   attach: function(context) {
     if (!Backdrop.featureDetect.flexbox()) {
-      $('head').append('<link rel="stylesheet" type="text/css" href="/core/modules/layout/css/grid-float.css">');
+      $('body').once('flexbox-test', function(){
```

Comment:
Weird... added that to make sure the file didn't get added multiple on a page in case of AJAX events. I'll try a different approach.